source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm

```diff
@@ -154,7 +190,9 @@ - (void)pushCardContext:(ACOAdaptiveCard *)card
     auto crtl = adaptiveCard->GetRtl();
     NSNumber *key = [NSNumber numberWithLong:(adaptiveCard->GetInternalId()).Hash()];
     BOOL hasSelectAction = adaptiveCard->GetSelectAction() ? YES : NO;
-    [self addToANewContext:crtl key:key hasSelectAction:hasSelectAction];
+    NSNumber *iconPlacement = [self.hostConfig getIconPlacement:key];
+
+    [self addToANewContext:crtl key:key hasSelectAction:hasSelectAction iconPlacement:iconPlacement];
```

Comment:
Same question here

Reply:
No, it won't. The function does check its nullability. If it's null, the icon placement will tell the caller that AboveTitle enum value won't get used.